src/content-scripts/message-tracking/debris-field-report-tracking.ts

```diff
@@ -131,41 +121,41 @@ function onMessage(message: Message<MessageType, any>) {
     }
 }
 
-function trackDebrisFieldReports(lang: LanguageKey, elem: Element) {
-    const messages = Array.from(elem.querySelectorAll('li.msg[data-msg-id]'))
-        .filter(elem => !elem.classList.contains(cssClasses.messages.base));
-    const headers = messageHeaders[lang];
+function trackDebrisFieldReports(elem: Element) {
+    const tabLabel = document.querySelector(`.tabsWrapper .innerTabItem.active[data-subtab-id="${tabIds.misc}"]`);
+    if (tabLabel == null) {
+        return;
+    }
+
+    const messages = Array.from(elem.querySelectorAll('div.msg[data-msg-id]'))
+        .filter(elem => {
+            const messageType = elem.querySelector('.rawMessageData')?.getAttribute('data-raw-messagetype');
+            return !elem.classList.contains(cssClasses.messages.base) && messageType === "32"; // @WONKY maybe created a typed version for messageTypes 
+        });
 
     messages.forEach(msg => {
         const id = parseIntSafe(msg.getAttribute('data-msg-id') ?? _throw('Cannot find message id'), 10);
 
         try {
             // prepare message to service worker
-            const dateText = msg.querySelector('.msg_head .msg_date')?.textContent ?? _throw('Cannot find message date');
-            const date = parse(dateText, dateTimeFormat, new Date()).getTime();
-            if (isNaN(date)) {
-                _throw('Message date is NaN');
-            }
+            const attributes = getMessageAttributes(msg, 'rawMessageData');
+            _logDebug(attributes);
 
-            const messageTextElem = msg.querySelector('.msg_content') ?? _throw('Cannot find message content element');
-            const text = messageTextElem.textContent ?? '';
-            const html = messageTextElem.innerHTML;
-
-            const messageTitle = msg.querySelector('.msg_title')?.textContent ?? '';
-            if (!messageTitle.toLowerCase().includes(headers.debrisField.toLowerCase())) {
-                //ignore anything that's not a DF harvest report
-                return;
+            const timestamp = attributes["timestamp"] ?? _throw('Cannot find message timestamp');
+            const date = parseInt(timestamp, 10) * 1000;
+            if (isNaN(date)) {
+                _throw('Message timestamp is NaN');
             }
-
+            
             // send message to service worker
+            // @WONKY are you more willing to keep attributes all togheter or to update the message data to separate every required attribute

```

Comment:
I would prefer to have a type each instead of sending all the attributes. This is way it's clear what the messages need, and we can detect errors before sending to the background worker.
But it doesn't make a huge difference, so I'm fine with this solution  as well